source/codegen/metadata/nirfmxbluetooth/config.py

```diff
@@ -6,7 +6,7 @@
     'service_class_prefix': 'NiRFmxBT',
     'java_package': 'com.ni.grpc.nirfmxbt',
     'csharp_namespace': 'NationalInstruments.Grpc.NiRFmxBT',
-    'namespace_component': 'nirfmxbt',
+    'namespace_component': 'nirfmxbluetooth',
```

Comment:
I think you'll also need to update 'service_class_prefix', 'driver_name', 'java_package', 'csharp_namespace'.